hknweb/events/templates/events/show_details.html

```diff
@@ -32,6 +32,13 @@ <h1 id="event-detail-title"> {{ event.name }} </h1>
     <p><b>Location</b>: {{ event.location }}</p>
     <p><b>Date</b>: {{event.start_time|date:"D, N j, Y"}}</p>
     <p><b>Time</b>: {{event.start_time|time}} to {{event.end_time|time}}</p>
+
+    {% if can_edit %}
+      <p><a href="{% url 'events:edit' event.id %}">
+          Edit This Event
```

Comment:
Is title capitalization appropriate? `Edit this event` is consistent with the `Add an event` capitalization on /events anyways.

Reply:
Uhhh I like this capitalization style

Reply:
Besides "Go Back to Calendar" is capitalized